src/main/resources/l10n/JabRef_da.properties

```diff
@@ -950,6 +952,8 @@ Reference_library=Referencelibrary
 
 Refine_supergroup\:_When_selected,_view_entries_contained_in_both_this_group_and_its_supergroup=Undergruppe\:_Vis_poster_indeholdt_både_i_denne_gruppe_og_gruppe_over
 
+Regex=
```

Comment:
Regex is the same as regular expression, I would reuse that existing translation